### PR TITLE
SEO-180458-Xamarin-ug-redirect-links

### PR DESCRIPTION
**Pull request by Edithabich**
Hi @MallikaRK ,
Kindly review the changes.

Regards,
Edith.

### CODE REVIEW

File: Xamarin/Maps/MapsProvider.md [hunk 1]
```diff
@@ -313,7 +313,7 @@ The detailed explanation of marker and its customization have been provided in M
 
 ## Load custom map
 
-You can load the custom map providers map such as Google Maps, TomTom using the [`RequestTileUri`](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ImageryLayer.html) event of ImageryLayer. For more details about  [`RequestTileUri`](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ImageryLayer.html), refer the [`Events`](https://help.syncfusion.com/xamarin/maps/mapsprovider#events) of ImageryLayer.
+You can load the custom map providers map such as Google Maps, TomTom using the [`RequestTileUri`](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ImageryLayer.html) event of ImageryLayer. For more details about  [`RequestTileUri`](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ImageryLayer.html#Syncfusion_SfMaps_XForms_ImageryLayer_RequestTileUri), refer the [`Events`](https://help.syncfusion.com/xamarin/maps/mapsprovider#events) of ImageryLayer.
```
Comment (Yvone-Atieno) [inline Xamarin/Maps/MapsProvider.md:316]: Update the correct link
image

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: Xamarin/Maps/MapsProvider.md (hunk 1 above)
Comment (Edithabich) [reply Xamarin/Maps/MapsProvider.md]: resolved

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]

**Pull request merged**